### ISSUE TITLE
Fracture blister

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7600]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3088393/

remove filter from phenopacket CSV after term added

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
done
